.eslintrc.json

```diff
@@ -76,7 +76,7 @@
     "vue/multi-word-component-names": [
       "error",
       {
-        "ignores": ["Modal", "Tabs", "Toast", "Login", "Navbar", "Settings"]
+        "ignores": ["Modal", "Tabs", "Toast", "Login", "Navbar", "Settings", "index"]
```

Comment:
Is this change needed?

Reply:
Not really, it was for previous approach but is not required now.